javascripts/govuk/multivariate-test.js

```diff
@@ -76,14 +76,23 @@
   };
 
   MultivariateTest.prototype.setCustomVar = function(cohort) {
-    if (this.customDimensionIndex) {
-      GOVUK.analytics.setDimension(
-        this.customDimensionIndex,
-        this.cookieName() + "__" + cohort
-      );
+    if (this.customDimensionIndex &&
+      this.customDimensionIndex.constructor == Array) {
```

Comment:
Sorry for the nit picking but we could do `===` instead of `==` here which.

(Once we have proper linting I won't have to make these annoying comments sorry :P)